src/api/database/changes.rst

```diff
@@ -685,7 +685,7 @@ amount of duplicated code.
     While :ref:`map functions <mapfun>` doesn't process the design documents,
     using ``_view`` filter forces them to do this. You need to be sure, that
     they are ready to handle documents with *alien* structure without panic
-    crush.
+    crash.
```

Comment:
I'd just remove the word entirely.  "panic crash" is redundant.  I think "without panic." is better. What do you think?

Reply:
Agreed.